test/integration/component/test_redundant_router_cleanups.py

```diff
@@ -666,3 +667,156 @@ def test_network_gc(self):
                              "Router should be in stopped state"
                              )
         return
+
+    @attr(tags=["advanced", "advancedns"], required_hardware="false")
+    def test_restart_ntwk_MVR_destroyed(self):
```

Comment:
@DaanHoogland , I have modified the method name as per your review comment. Can you please review it?